src/main/scala/battleship/routes/gameActor.scala

```diff
@@ -0,0 +1,84 @@
+package battleship.routes
+
+import akka.actor.{Actor, ActorLogging, ActorRef, Props}
+import battleship.PlayerActor.{GetNextShot, PlaceBoats}
+import battleship.game._
+import battleship.routes.gameActor.{StartGame, ThisIsBoatSetup, ThisIsNextMove}
+
+/**
+  * Created by m06f947 on 3-8-2017.
+  */
+
+
+
+object gameActor {
+
+  case class StartGame(boardsize:Int, player1:ActorRef, player2:ActorRef, boatset: Seq[Boat])
+
+  //case class WhatIsBoatSetup(boats:Seq[Boat], boardsize: Int)
+  case class ThisIsBoatSetup(placement: Set[(Boat,BoatLocation)])
+  //case class WhatIsNextMove(boardsize:Int, shotHistory: Seq[((Int,Int),ShotResult)])
+  case class ThisIsNextMove(x: Int, y: Int)
+
+
+  def props: Props = Props(new gameActor)
+
+}
+
+class gameActor extends Actor with ActorLogging {
+
+
+  //internal state
+  var boardsize = 8

```

Comment:
For the default values, you could also refer to them like `var boardsize = Game.defaultBoardSize` (same for boatSet in the next line)